src/main/resources/messages_en.properties

```diff
@@ -651,6 +651,23 @@ program.favorite.type.not.identified=The following favorite type {0} is not defi
 #Inventory Module
 location.required=Location is required
 location.invalid=Invalid location identifier
+location.is.used.in.lots=Location Id {0} can not be deleted because is being used in Lots
+location.is.used.in.names=The location Id {0} is used in germplasm names
+location.is.used.in.attributes=The location Id {0} is used in attributes
+location.is.used.in.studies=The location Id {0} is used in studies
+location.is.used.in.germplasms=The location Id {0} is used in germplasms
+location.abbr.is.in.used=The location Abbr is already used
+location.abbr.is.required=The location Abbr is required
+location.abbr.max.length=The location Abbr exceeds {0} length
```

Comment:
The location Abbr exceeds {0} length should be replaced by 
The location Abbr exceeds maximum length of {0} characters 
@cuenyad